nerfstudio/process_data/record3d_utils.py

```diff
@@ -87,6 +95,23 @@ def record3d_to_json(images_paths: List[Path], metadata_path: Path, output_dir:
 
     out["frames"] = frames
 
+    # If .ply directory exists add the sparse point cloud for gsplat point initialization
+    if ply_dirname is not None:
+        assert ply_dirname.exists(), f"Directory not found: {ply_dirname}"
+        assert ply_dirname.is_dir(), f"Path given is not a directory: {ply_dirname}"
+
+        # Create sparce point cloud
+        pcd = o3d.geometry.PointCloud()
+        for ply_filename in ply_dirname.iterdir():
+            temp_pcd = o3d.io.read_point_cloud(str(ply_filename))
+            pcd += temp_pcd.voxel_down_sample(voxel_size=0.8)
```

Comment:
Should probably make `voxel_size` an argument, and maybe include it in the ProcessRecord3D settings.

Reply:
thats a good idea